tests/rkt_run_pod_manifest_test.go

```diff
@@ -107,6 +107,27 @@ func TestPodManifest(t *testing.T) {
 		cgroup         string
 	}{
 		{
+			// Special characters
+			[]imagePatch{
+				{"rkt-test-run-pod-manifest-read.aci", []string{}},
+			},
+			&schema.PodManifest{
+				Apps: []schema.RuntimeApp{
+					{
+						Name: baseAppName,
+						App: &types.App{
+							Exec:  []string{"/inspect", "--print-msg=\n'\"$"},
+							User:  "0",
+							Group: "0",
+						},
+					},
+				},
+			},
+			true,
+			"'\"$",
+			"",
```

Comment:
When testing this, this freezes. I get:

```
# cd /tmp/datadir-931636457/pods/run/ecd24df2-815d-40fe-bd25-1d047ec766d8

# json_pp < pod
            "exec" : [
               "/inspect",
               "--print-msg=\n'\"$"
            ],

# grep -A 1 'ExecStart=' stage1/rootfs/usr/lib64/systemd/system/rkt-inspect.service
ExecStart="/appexec" "/opt/stage2/rkt-inspect/rootfs" "/" "/rkt/env/rkt-inspect" "0" "0" "/inspect" "--print-msg=
\'\"$$"

# nsenter -t 32672 -p -m systemctl status rkt-inspect.service
● rkt-inspect.service - Application=rkt-inspect Image=coreos.com/rkt-inspect
   Loaded: error (Reason: Bad message)
   Active: inactive (dead)

```

So the pod manifest is correct but the `.service` file has an unescaped new line, which causes the freeze.